cfme/common/vm_console.py

```diff
@@ -140,9 +140,11 @@ def send_keys(self, text):
         """Send text to the console."""
         self.switch_to_console()
         canvas = self.provider.get_remote_console_canvas()
-        canvas.send_keys(text)
-        canvas.send_keys(Keys.ENTER)
         logger.info("Sending following Keys to Console {}".format(text))
+        for character in text:
+            canvas.send_keys(character)
+            wait_for(lambda: False, delay=0.3, num_sec=0.3, silent_failure=True)
```

Comment:
@mfalesni , okay. Yes I did replace time.sleep because @quarckster  suggested it, but what you are saying is perfectly making sense. I will change it back to original. Thanks for review.